pkg/swagger/swagger.go

```diff
@@ -123,11 +124,30 @@ func Run(api, outputDir string) error {
 				property.Schema.ClientFlatten = true
```

Comment:
As discussed on a call:
* We should ignore/remove SystemData definition generated from go struct definition
* Add a comment somewhere explaining why we ignore this (e.g. common types, etc.)